bugwarrior/services/bitbucket.py

```diff
@@ -204,8 +204,10 @@ def get_annotations2(self, tag, issue, issue_obj, url):
         )
 
     def get_owner(self, issue):
-        tag, issue = issue
-        return issue.get('responsible', {}).get('username', None)
+        _, issue = issue
+        assignee = issue.get('assignee', None)
+        if assignee is not None:
+            return assignee.get('username', None)
```

Comment:
Link to the new API docs for the curious: https://confluence.atlassian.com/bitbucket/version-2-423626329.html#Version2-Issues